msvc/sdk-build.props

```diff
@@ -24,6 +24,10 @@
     </ProjectReference>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Condition="'$(BuildingFrameworksCore)' != 'true'" Include="WinObjC.Frameworks.UWP" Version="*" />
```

Comment:
ah, I assumed you meant to just include it without the condition.
Consider it moved.